src/Makefile

```diff
@@ -157,26 +166,26 @@ distclean: clean
 	find ../doc -type f \( -name \*\\.gz -o -name \*\\.1 \) -exec rm -f '{}' +
 
 installdirs:
-	install -d $(bindir) \
+	$(INSTALL_CMD) -d $(bindir) \
```

Comment:
From a quick look at [`install(1m)`](https://illumos.org/man/1m/install): If this is the source of incompatibility, can't we just `mkdir -p` here instead to do what GNU and other `install`s do.

Reply:
I kid y'all not, I looked over it all again today, edited install(1m) back in and and it seems to be working just fine. Guess I'll just remove that little kludge!